databricks/koalas/tests/test_groupby.py

```diff
@@ -2245,3 +2245,34 @@ def test_is_multi_agg_with_relabel():
 
         assert is_multi_agg_with_relabel(a="max") is False
         assert is_multi_agg_with_relabel(a_min=("a", "max"), a_max=("a", "min")) is True
+
+    def test_get_group(self):
+        pdf = pd.DataFrame(
+            [
+                ("falcon", "bird", 389.0),
+                ("parrot", "bird", 24.0),
+                ("lion", "mammal", 80.5),
+                ("monkey", "mammal", np.nan),
+            ],
+            columns=["name", "class", "max_speed"],
+            index=[0, 2, 3, 1],
+        )
+        kdf = ks.from_pandas(pdf)
+
+        self.assert_eq(
+            kdf.groupby("class").get_group("bird"),
+            pdf.groupby("class").get_group("bird"),
+            check_exact=False,
```

Comment:
Why `check_exact=False`?

Reply:
Otherwise the `AssertionError` is raised shown as below.

```shell
AssertionError: check_exact may only be used with numeric Series

Left:
     name class  max_speed
0  falcon  bird      389.0
2  parrot  bird       24.0
name          object
class         object
max_speed    float64
dtype: object

Right:
     name class  max_speed
0  falcon  bird      389.0
2  parrot  bird       24.0
name          object
class         object
max_speed    float64
dtype: object
```

Reply:
It passes in my local.

Reply:
Could you let me know your test environment ?

In my case, `pandas==1.1.0`, `pyspark==3.0.1`, `python==3.8` and still failing 😿

Reply:
Oh, it works in `pandas==1.0.0`. maybe we should fix this.

Reply:
I tried it with pandas 1.1.2 as well and it worked.
maybe the bug was introduced in 1.1.0 and fixed in 1.1.1 or 1.1.2?

Reply:
yeah, It seems to be fixed in pandas 1.1.1. 

Thanks, @ueshin , I think we can just remove the `check_exact` :)